src/cryptocom/exchange/private.py

```diff
@@ -166,10 +166,12 @@ async def create_order(
         data['time_in_force'] = force_type.value
         data['exec_inst'] = exec_type.value
 
-        quantity = "{:.{}f}".format(quantity, pair.quantity_precision)
+        # here is the change i propose
```

Comment:
Nice! Look you can do this not duplicating formating string:
```
quantity_precision = pair.quantity_precision
if ...
    quantity_precision = pair.price_precision
quantity = "{:.{}f}".format(quantity, quantity_precision )
data['quantity'] = quantity
```

Remove comment "#", add version 0.10.1 into README and change __version__ variable and I will accept pull and will release new version)

Reply:
I removed the comment.

Reply:
and added version 0.10.1 into README and changed version __inti__.py

sorry, I'm new in github and i don't know if i did correctly.